### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by BenjaminMatembe**
On this feature I:

- Added the HTML and CSS rules
- Made sure all is generated and served by webpack

### CODE REVIEW

**Review by didierganthier — COMMENTED** [review]
Hi @BenjaminMatembe ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project that doesn't have a professional README file. Please refer to this [template](https://github.com/microverseinc/readme-template) to create the README. 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,208 @@
...
+Install node_modules dependencies by:
+
+>running npm install or yarn install
+
+### Usage
+
+To run the project, execute the following command:
+
+>right click on index.html and open with live server
+
```
Comment (LionRouge1) [inline README.md]: - [ ] Kindly update this section with the appropriate information about how to run the project.

File: README.md [hunk 2]
```diff
@@ -1 +1,208 @@
...
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Benjamin**
+
+- GitHub: [@githubhandle](https://github.com/BenjaminMatembe)
```
Comment (LionRouge1) [inline README.md:152]: [OPTIONAL]
- [ ] Kindly add your Linkedin and Twitter account to the README file. That will make easier to visitors or recruiters to reach out to you.

File: src/modules/loader.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const ulList = document.querySelector('.createdList');
+
+const list = [
+  { description: ['wash cups', 'goods for sale', 'clean the house', 'yello yellow'] },
+  { completed: [false, false, false, false] },
+  { index: [1, 2, 3, 4] },
+
+];
```
Comment (LionRouge1) [inline src/modules/loader.js]: - [ ] Nice work populating the tasks using array of object :+1: However It's required that each task object should contain three keys: `description [string]`, `completed [bool]`, and `index: [number]`. This array could look like this:
```sh
const list = [
  {
    description: 'A description about task 1',
    completed: false,
    index: 1,
  },
  {
    description: 'A description about task 2',
    completed: false,
    index: 2,
  },
  ...
];
```

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @BenjaminMatembe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR :white_check_mark: 
- Good commit :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @BenjaminMatembe 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Comment by BenjaminMatembe** [comment]
Hi @didierganthier Thanks for the advise given about the README, @LionRouge1 thanks for the requested changes for they opened my eye about the flow of the functionality, @AlphaNtihinduka thank you for approving my project for it gave me more motivation to proceed

**Pull request merged**